convert2rhel/actions/__init__.py

```diff
@@ -137,7 +183,7 @@ class Convert2rhelLatest(Action):
         """
 
     #: Override dependencies with a Sequence that has other :class:`Action`s
-    #: that must be run before this one.
+    #: :attr:`Action.id`s that must be run before this one.
```

Comment:
Would be nice to specify that the contents of `depdencies` should be in string format, not the class name. 

I think that people can look at it and have the wrong interpretation if they see like that, for instance, if I saw that I would do 

```python
dependencies = (Something.id, )
```

Rather than what we have right now.

Reply:
<nod>  `Something.id is technically correct yeah, but it has the problem of listing the same Action class twice that we ran into before.  I could change the code to automatically remove duplicate Action classes which would make both of these work without issues:

``` python
dependencies = ("SOMETHING",)
dependencies = (Something.id,)
```

That kinda seems like it might be the right thing to do?  What do you think?

Reply:
`Something.id` is simply a constant string value which happens to be defined inside of a class.  It feels like using a string literal or a variable that points to a constant string should both be legal here.

I resisted uniquifying the classes at first because I worried that someone might use the same class name twice and the uniquifying code might make that silently pass and be hard to debug.  However,

(1) the Action.id would still have to be unique (we have a unittest which will fail if there are duplicates) and
(2) the Action name would have to be repeated in the same file.  Python will return different Actions if two classes use the same name, just in different files.  So my original worry is probably not valid.

Should there be a best practice of using a string literal or the id inside of the Action's class?  Both have drawbacks... The string literal could be typoed (but we have a unittest which will fail if that happens).  Using Action.id requires importing the Action which is a bit more work for the programmer and for people reading the code and for a bit more work for the computer since it will detect the imported Action and then have to remove it since it is a duplicate.

Reply:
I've added code to uniqueify this and changed the docstring to make it clear that a string literal can be used here.  Is that okay?

Reply:
That's sounds good to me.